Userland/Applications/Browser/Tab.cpp

```diff
@@ -42,6 +42,11 @@
 
 namespace Browser {
 
+Tab::~Tab()
+{
+    clear_inspector_callbacks();
```

Comment:
Hmm, if we close the tab being inspected, feels like we should also just close the inspector window?

Reply:
Yep, that seems more sensible (probably other use-after-frees from keeping them around). I now also close the other inspectors (JS console, cookie, etc).